ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1516,7 +1520,7 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
                 case MAV_FRAME_GLOBAL:
                 case MAV_FRAME_GLOBAL_INT:
                 default:
-                    // Copter does not support navigation to absolute altitudes. This convert the WGS84 altitude
+                    // Copter does not support navigation to absolute altitudes. This converts the WGS84 altitude
```

Comment:
By the way, copter doesn't actually support absolute altitudes now so the comment is a bit out of date.

Reply:
Randy just type in what that comment should look like, and I will change it.

BTW the mavlink specification call for a AMSL altitude here, but it looks like the current implementation is using WGS84.
Am I missing something here ?